Bash/fizzbuzz.sh

```diff
@@ -1,13 +0,0 @@
-# Hello World Program in Bash Shell
```

Comment:
You need to revert this file. Use:
git checkout 5198bc0bf4cb885fa5ca58e5836be19e98abd22c Bash/fizzbuzz.sh
and then commit this modification
git commit

For details visit: https://stackoverflow.com/questions/2733873/reverting-a-single-file-to-a-previous-version-in-git